### PR TITLE
Ivan bankaccounts

### PR DESCRIPTION
**Pull request by ivankraev**
# Create Api Endpoints for Bankaccounts model.

- Post request - inserts entity in the database
- Get request - returns all records for the entity from the database
- Get by id - returns unique record selected by id
- Patch request - updates unique entity selected by id
- Delete request - deletes unique record selected by id
- Post request - deletes one or many records selected by id's in array

## Post request
pullpost
## Get request 
pull1
## Get by id 
pullgetbyid
## Patch request 
pullpatch
## Delete request
pulldelete
## Delete Many
pulldeletemany

### CODE REVIEW

File: db/seed/bankaccount.seed.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { PrismaClient } from '@prisma/client'
+const prisma = new PrismaClient()
+
+export async function bankAccountsSeed() {
+  console.log('Cities seed')
+
+  const insert = await prisma.bankAccount.createMany({
+    data: [
+      {
+        accountHolderName: 'Georgi Pavlov',
```
Comment (kachar) [inline db/seed/bankaccount.seed.ts:10]: You can use random data so we're not using real names by accident

File: db/seed/index.ts [hunk 2]
```diff
@@ -25,6 +26,7 @@ async function main() {
       await coordinatorSeed(),
       await beneficiarySeed(),
       await campaignSeed(),
+      await bankAccountsSeed(),
```
Comment (kachar) [inline db/seed/index.ts:29]: Let's make this method singular just like the other sibling functions
```suggestion
      await bankAccountSeed(),
```

File: migrations/20220201085056_pulling_data/migration.sql [hunk 3]
```diff
@@ -0,0 +1 @@
+-- This is an empty migration.
```
Comment (kachar) [inline migrations/20220201085056_pulling_data/migration.sql:1]: Looks like we don't need this migration

**Review by kachar — CHANGES_REQUESTED** [review]
@ivankraev Nice iteration! 

Let's do some more work to make sure the bank accounts are not leaked to public and we'll merge it soon

File: migrations/20220201085056_pulling_data/migration.sql (hunk 3 above)
Comment (ivankraev) [reply migrations/20220201085056_pulling_data/migration.sql:1]: Thanks for the review ! I will fix the mentioned problems am update the pull request ASAP :)

**Review by kachar — APPROVED** [review]

File: apps/api/src/bankaccount/dto/create-bankaccount.dto.ts [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  @Expose()
+  @IsString()
+  @IsOptional()
+  bankName: string | null
+
+  @ApiProperty()
+  @Expose()
+  @IsString()
+  @IsOptional()
+  bankIdCode: string | null
```
[the remark this answers is not in the document]
Comment (kachar) [reply apps/api/src/bankaccount/dto/create-bankaccount.dto.ts:39]: @ivankraev Please add the validations to the entity in an additional PR as this one will be merged now.

**Pull request merged**